src/mscorlib/src/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -357,7 +357,7 @@ private unsafe static bool IsSortable(char *text, int length)
         // ---- PAL layer ends here ----
         // -----------------------------
 
-        internal unsafe int GetHashCodeOfStringCore(string source, CompareOptions options, bool forceRandomizedHashing, long additionalEntropy)
+        internal unsafe int GetHashCodeOfStringCore(string source, CompareOptions options, long additionalEntropy)
```

Comment:
The additionalEntropy argument looks unused. It does not exist on Windows. I think it can be deleted.